docs/translations.md

```diff
@@ -0,0 +1,44 @@
+# Translations
+
+We are *delighted* that people are willing to work to translate materials
+into various languages such as Japanese. Here is how we review translation PRs
+in this repo to ensure translations are good ones.
+
+## Translation review process
+
+To ensure that translations are good ones:
+
+1. All pull requests (PRs) that add or modify a translation
+   *must* be labeled as such.
+   The label is the language name, e.g., "Japanese" for Japanese.
+   Anyone can add that label, including the originator,
+   so that missing labels can be easily added.
```

Comment:
Can we verify that this is actually possible for any contributor? You probably need to be a member of the ossf org and/or have write permissions for this repo.

Reply:
I meant "is allowed to by the process", not "GitHub permits this". If you're an originator, and you've added the PR in your own repo, I'm pretty sure you can add a label in your own repo.

Reply:
It's a fair point. I've tweaked the proposed text to clarify that case.

Reply:
Thanks, David. And in the meantime also I realized that I may have overcomplicated this in my head: with "adding a label", I was thinking of adding a new label to the list of labels, i.e., managing the labels not just adding an existing label to a PR. Anyhow, we absolutely don't need to over-complicate this. This updated version is ok for me.